extensions/notion/src/utils/notion/database/property.ts

```diff
@@ -19,7 +19,7 @@ export function formatDatabaseProperty(...[type, value]: FormatDatabasePropertyP
     case "date": {
       if (!value) return;
       type DatePropertyTimeZone = Exclude<Extract<PageProperty, { type?: "date" }>["date"], null>["time_zone"];
-      const time = subMinutes(new Date(value), new Date().getTimezoneOffset()).toISOString().split("T")[0];
+      const time = subMinutes(new Date(value), new Date().getTimezoneOffset()).toISOString();
```

Comment:
This looks a bit weird to me. Why was `.split("T")[0]` used before? Are we sure we're not adding another regression with this fix?